lib/tapioca/gem/events.rb

```diff
@@ -42,6 +42,23 @@ def initialize(symbol, constant)
       end
     end
 
+    class NotOwnedConstantFound < Event
```

Comment:
What if this subclassed `ConstantFound`? We are basically using the class as a discriminator on how to act, otherwise, all aspects of this class should be the same as `ConstantFound`.